packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -1995,10 +1995,10 @@ class AnimatedPhysicalModel extends ImplicitlyAnimatedWidget {
   const AnimatedPhysicalModel({
     super.key,
     required this.child,
-    required this.shape,
+    this.shape = BoxShape.rectangle,
     this.clipBehavior = Clip.none,
-    this.borderRadius = BorderRadius.zero,
-    required this.elevation,
+    this.borderRadius,
```

Comment:
Why remove the explicit default value for borderRadius?

Reply:
The non-animated class has a nullable border radius, so this change makes them consistent.